src/js/veteran-representative/config/form.js

```diff
@@ -0,0 +1,274 @@
+  authorizationToChangeClaimantAddress,
+} = fullSchema.properties;
+
+const { fullName } = fullSchema.definitions;
+
+const authorizationForRepresentativeAccessToRecordsDescription =
+  "I authorize the VA facility having custody of my VA claimant records to disclose to the service organization named in Item 3A treatment records relating to drug abuse, alcoholism or alcohol abuse, infection with the human immunodeficiency virus (HIV), or sickle cell anemia. Redisclosure of these records by my service organization representative, other than to the VA or the Court of Appeals for Veterans Claims, is not authorized without my further consent. This authorization will remain in effect until the earlier of the following events: (1) I revoke this authorization by filing a written revocation with VA; or (2) I revoke the appointment of the service organization named above, either by explicit revocation or the appointment if another representative";
+
+const limitationOfConsentDescription =
+  "I authorize disclosure of records related to treatment for all conditions listed in Item 12 except:";
+
+const authorizationToChangeClaimantAddressDescription =
+  "I authorize any official representative of the organization named in Item 3A to act on my behalf to change my address in my VA records. This authorization does not extend to any other organization without my further written consent. This authorization will remain in effect until the earlier of the following events: (1) I file a written revocation with VA; or (2) I appoint another representative, or (3) I have been determined unable to manage my financial affairs and the individual or organization named in Item 3A is not my appointed fiduciary";
+
+const formConfig = {
+  urlPrefix: "/",
+  submitUrl: "/v0/api",
+  trackingPrefix: "form-2122-",
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: "21-22",
+  version: 0,
+  prefillEnabled: true,
+  savedFormMessages: {
+    notFound: "Please start over to apply for veteran representative.",
+    noAuth:
+      "Please sign in again to continue your application for veteran representative.",
+  },
+  title: "Appoint a Veteran Service Officer as your representative",
+  defaultDefinitions: {
+    fullName,
+  },
+  chapters: {
+    veteranInformation: {
+      title: "Veteran Information",
+      pages: {
+        veteranInformation: {
+          path: "veteran-information",
+          title: "Veteran information",
+          uiSchema: {
+            fullName: fullNameUI,
+            veteranSSN: ssnUI,
+            vaFileNumber: { "ui:title": "VA file number" },
+            insuranceNumber: { "ui:title": "Insurance Number" },
+          },
+          schema: {
+            type: "object",
+            required: [
+              'fullName',
+              'veteranSSN',
+              'vaFileNumber',
+              'insuranceNumber',
+            ],
+            properties: {
+              fullName,
```

Comment:
I think you want this to be `veteranFullName`, since `fullName` is the name of the common definition but not the actual field in the schema.